@here/harp-mapview-decoder/lib/TileDataSource.ts

```diff
@@ -152,6 +156,9 @@ export class TileDataSource<TileType extends Tile> extends DataSource {
         });
 
         this.cacheable = true;
+        this.m_tileLoaderCache = new LRUCache<number, TileLoader>(40, tileLoader => {
```

Comment:
40 seems quite big for wrap-around tiles.

Reply:
correct, this is the number used in VisibleTileSet for the tile cache, I have made it a bit clearer.